portfolio/src/main/webapp/script.js

```diff
@@ -89,3 +89,13 @@ function checkLoginStatus() {
     }
   });
 }
+
+/**
+ * Renders the map in the assigned div tag.
+ */
```

Comment:
nit: What kind of map? What is the map? Consider elaborating a bit more on this comment. You could include the fact that it is using the Google Maps API and also what the intent of the map is (ex. showing your home town, showing your university, showing your favorite restaurants, etc.)

Reply:
Done!